landing_page/spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+    context "create a new user account with acceptable credentials" do
+
+        it "successfully adds a new user" do
+            expect{create_new_user}.to change(User, :count).by(1)
+        end
+
+        it "logs the user in upon account creation" do
+            create_new_user
+            expect(session[:user_id]).to_not be_nil
+        end
+
+        it "redirects a new user to their profile page upon creation" do
+            create_new_user
+            expect(response).to redirect_to(assigns(:user))
+        end
+
+    end
+
+    context "GET #index" do
+
+        it "shows the list of users, expects a redirect response to be 200" do
+            get :index
+            expect(response.status).to eq(200)
+        end
+
+    end
+
+    context "GET #show" do
+
+        it "shows a user their profile, expect redirect response to be 200" do
```

Comment:
The scenario description should not have the expectation.

```suggestion
        it "shows a user their profile" do
```

Additionally, this expectation part is wrong, this scenario does not involve a redirect. With that in mind, a couple os questions:

1. What do you mean with "redirect response"?
1. What does the `200` mean? What is it?

Reply:
They overlap. So the 200 is to show that the redirect response was a success. This has since been altered in commit fa0508cf969ee02dc88928e9c412d18df6ea8a74